src/app/core/config/models/config-submission-access.model.ts

```diff
@@ -0,0 +1,27 @@
+import { autoserialize, inheritSerialization, deserialize } from 'cerialize';
+import { typedObject, link } from '../../cache/builders/build-decorators';
+import { ConfigObject } from './config.model';
+import { AccessesConditionOption } from './config-accesses-conditions-options.model';
+import { SUBMISSION_ACCESSES_TYPE } from './config-type';
+import { HALLink } from '../../shared/hal-link.model';
+
+
+@typedObject
+@inheritSerialization(ConfigObject)
+export class SubmissionAccessModel extends ConfigObject {
+  static type = SUBMISSION_ACCESSES_TYPE;
+  /**
+   * A list of available bitstream access conditions
```

Comment:
This comment is not entirely correct...these are no longer *bitstream* specific access conditions...they may also apply to the entire Item.